### PR TITLE
[SecurityBundle] Fix 4.3 tests forward compat

### PR DESCRIPTION
**Pull request by yceruto**
| Q             | A
| ------------- | ---
| Branch?       | 4.3
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | https://github.com/symfony/symfony/issues/33478
| License       | MIT
| Doc PR        | -

### CODE REVIEW

**Review by derrabus — APPROVED** [review]

**Comment by nicolas-grekas** [comment]
What about the BC break part @derrabus mentions in the linked issue? I didn't try to understand it, what about you? :)

**Comment by yceruto** [comment]
> What about the BC break part @derrabus mentions in the linked issue? I didn't try to understand it, what about you? :)

It's not a BC break to me, but a side effect of the previous behavior related to PHPUnit and error handlers. See https://github.com/symfony/symfony/issues/33478#issuecomment-528441668

File: src/Symfony/Bundle/SecurityBundle/Tests/Functional/app/MissingUserProvider/config.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 imports:
-    - { resource: ./../config/framework.yml }
+    - { resource: ./../config/default.yml }
```
Comment (Tobion) [inline src/Symfony/Bundle/SecurityBundle/Tests/Functional/app/MissingUserProvider/config.yml]: should that be reverted in 4.4 again?

**Review by Tobion — APPROVED** [review]

File: src/Symfony/Bundle/SecurityBundle/Tests/Functional/app/MissingUserProvider/config.yml (hunk 1 above)
Comment (yceruto) [reply src/Symfony/Bundle/SecurityBundle/Tests/Functional/app/MissingUserProvider/config.yml]: Yap!

**Review by nicolas-grekas — APPROVED** [review]

**Comment by nicolas-grekas** [comment]
Thank you @yceruto.

**Pull request merged**